src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,146 @@
+
+    private int calculateMatchedLottoAutomaticTicket(
+            LottoAutomaticTicket lottoAutomaticTicket,
+            LastWeekWinningLotto lastWeekWinningLotto) {
+
+        List<Integer> lottoTicket = lottoAutomaticTicket.getLotto();
+        List<Integer> lastWeekWinningLottoTicket = lastWeekWinningLotto.getLotto();
+        int matchedCount = 0;
+        for (int lottoNumber = 0; lottoNumber < LOTTO_SIZE; lottoNumber++) {
+            Boolean isMatchLottoNumber = lottoTicket.contains(lastWeekWinningLottoTicket.get(lottoNumber));
+            matchedCount += countMatchedLottoNumber(isMatchLottoNumber);
+        }
+        return matchedCount;
+
+    }
+
+    private Boolean calculateMatchedBonusBallManualTicket(
+            LottoManualTicket lottoManualTicket,
+            LastWeekWinningBonusBall lastWeekWinningBonusBall) {
+
+        List<Integer> lottoTicket = lottoManualTicket.getLotto();
+        int WinningBonusBalls = lastWeekWinningBonusBall.getLastWeekWinningBonusBall();
+        return lottoTicket.contains(WinningBonusBalls);
+    }
+
+    private int calculateMatchedLottoManualTicket(
+            LottoManualTicket lottoManualTicket,
+            LastWeekWinningLotto lastWeekWinningLotto) {
+
+        List<Integer> lottoTicket = lottoManualTicket.getLotto();
+        List<Integer> lastWeekWinningLottoTicket = lastWeekWinningLotto.getLotto();
+        int matchedCount = 0;
+        for (int lottoNumber = 0; lottoNumber < LOTTO_SIZE; lottoNumber++) {
+            Boolean isMatchLottoNumber = lottoTicket.contains(lastWeekWinningLottoTicket.get(lottoNumber));
+            matchedCount += countMatchedLottoNumber(isMatchLottoNumber);
+        }
+        return matchedCount;
+    }
+
+    private int countMatchedLottoNumber(Boolean isMatchLottoNumber) {
+        if (isMatchLottoNumber) {
+            return 1;
+        }
+        return 0;
+    }
+
+    public List<WinningStatus> getStatistics(List<LottoMatchStatus> lottoAllTicketMatchStatuses) {
+        for (LottoMatchStatus lottoMatchStatus : lottoAllTicketMatchStatuses) {
+            mappingLottoTicketWithBonusBall(lottoMatchStatus);
+        }
+        return lottoPrices;
+    }
+
+    private void mappingLottoTicketWithBonusBall(LottoMatchStatus lottoMatchStatus) {
+        for (WinningStatus winningStatus : WinningStatus.values()) {
+            compareWinningStatus(winningStatus, lottoMatchStatus.getMatchedCount(), lottoMatchStatus.getHasBonusBall());
+        }
+    }
+
+    private void compareWinningStatus(WinningStatus winningStatus, int lottoMatchedCount, Boolean hasBonusBall) {
+        if ((winningStatus.getMatchCount() == lottoMatchedCount) && (lottoMatchedCount != WINNING_LOTTO_WITH_BONUS_BALL_MATCHED_COUNT)) {
+            lottoPrices.add(winningStatus);
+        }
+        if ((winningStatus.hasBonusBall() == hasBonusBall) && (lottoMatchedCount == WINNING_LOTTO_WITH_BONUS_BALL_MATCHED_COUNT)) {
+            lottoPrices.add(winningStatus);
+        }
+    }
```

Comment:
```suggestion
    private void compareWinningStatus(WinningStatus winningStatus, int lottoMatchedCount, Boolean hasBonusBall) {
        if (매치 카운트가 5개일 경우 || 매치 카운트가 5개가 아닐 경우) {
            lottoPrices.add(winningStatus);
        }
    }
```
if문 안을 메서드로 분리하여 이렇게 통일할 수 있겠어요